README.md

```diff
@@ -1,2 +1,97 @@
-# notifications-ruby-client
-Ruby client for notifications API
+# GOV.UK notifications-ruby-client
+
+Ruby client for the GOV.UK Notifications API
```

Comment:
Just on a stylistic note, we should probably be talking about GOV.UK Notify rather than GOV.UK Notifications (see https://github.com/alphagov/notifications-java-client).